yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -93,10 +94,10 @@ func runImportDataStatusCmd() error {
 			uiTable.AddRow(row.tableName, row.fileName, row.status, totalCount, importedCount, perc)
 		} else {
 			if i == 0 {
-				addHeader(uiTable, "TABLE", "FILE", "STATUS", "TOTAL ROWS", "IMPORTED ROWS", "PERCENTAGE")
+				addHeader(uiTable, "TABLE", "STATUS", "TOTAL ROWS", "IMPORTED ROWS", "PERCENTAGE")
```

Comment:
For import data file, we still show file, correct?

Reply:
Right